src/OpenLoco/Window.h

```diff
@@ -267,7 +267,7 @@ namespace OpenLoco::Ui
         int16_t rowInfo[1000];                             // 0x6A
         uint16_t rowCount;                                 // 0x83A
         uint16_t var_83C;
-        uint16_t rowHeight;    // 0x83E
+        uint16_t kRowHeight;   // 0x83E
```

Comment:
Accidental rename?

Reply:
yep, looks like VS renamed this one as well as the other variables with the same name which were constants